lib/beaker/options/parser.rb

```diff
@@ -67,29 +60,39 @@ def split_arg arg
       #                         no .rb files are found overall
       def file_list(paths)
         files = []
-        if not paths.empty?
+        unless paths.empty?
```

Comment:
We don't have a style guide here, so there's no objective way to rule on this. In style guides I've lived under in the past, we had a couple of things that could apply here:

 - "Don't use the block form of `unless`" (i.e., only use inline `unless`)
 - "Always use `!` instead of `not`"

No change needed, just highlighting this and our continued lack of a real ruby style guide.